lib/graphhopper/models/activity.rb

```diff
@@ -77,7 +77,7 @@ def initialize(attributes = {})
 
     # Custom attribute writer method checking allowed values (enum).
     def type=(type)
-      allowed_values = ["start", "end", "service", "pickupShipment", "deliverShipment"]
+      allowed_values = ["start", "end", "service", "pickupShipment", "deliverShipment", "pickup", "delivery"]
```

Comment:
I'll not block it now but those should not be in an array of local values imo but in a constant ;P